test/Integration.purs

```diff
@@ -42,6 +43,7 @@ testPlan = do
   Collateral.suite
   PrivateKey.suite
   Logging.suite
+  NetworkId.suite
```

Comment:
Please let's move this test to plutip tests. The reason for this is that integration tests are not run automatically, while this test definitely can.